src/Assert.php

```diff
@@ -438,9 +489,11 @@ public static function isAOf(mixed $value, mixed $class, string $message = ''):
      * @param object|string $value
      * @psalm-param class-string<UnexpectedType> $class
      *
+     * @psalm-return !UnexpectedType
+     *
      * @throws InvalidArgumentException
      */
-    public static function isNotA(mixed $value, mixed $class, string $message = ''): void
+    public static function isNotA(mixed $value, mixed $class, string $message = ''): mixed
```

Comment:
While `: mixed` is the correct return, the expectation is that `$value` is `string|object`. Perhaps we need an `objectish($value)` method? As in:

```php
// Check that value is an object or class-string
static::objectish($value);
```